sdcm/tester.py

```diff
@@ -1185,7 +1185,7 @@ def cql_connection_patient(self, node, keyspace=None,  # pylint: disable=too-man
 
     @retrying(n=8, sleep_time=15, allowed_exceptions=(NoHostAvailable,))
     def cql_connection_patient_exclusive(self, node, keyspace=None,  # pylint: disable=invalid-name,too-many-arguments,unused-argument
-                                         user=None, password=None, timeout=30,
```

Comment:
@aleksbykov why you've removed this timeout?

Reply:
Because it cause an error when kwargs unpacked

Reply:
It is used in other places in the code
![image](https://user-images.githubusercontent.com/5269241/84600436-e12c1c00-ae81-11ea-8cd1-3995b05a26d9.png)

need to remove from there

Reply:
fixed.